application/src/main/java/org/togetherjava/tjbot/commands/help/HelpSystemHelper.java

```diff
@@ -333,6 +333,55 @@ private void executeUncategorizedAdviceCheck(long threadChannelId, long authorId
         }).queue();
     }
 
+    void scheduleNoActivityAdviceCheck(long threadChannelId, long authorId) {
+        SERVICE.schedule(() -> {
+            try {
+                executeNoActivityAdviceCheck(threadChannelId, authorId);
+            } catch (Exception e) {
+                logger.warn(
+                        "Unknown error during a no activity advice check on thread {} by author {}.",
+                        threadChannelId, authorId, e);
+            }
+        }, SEND_NO_ACTIVITY_ADVICE_AFTER_MINUTES, TimeUnit.MINUTES);
+    }
+
+    private void executeNoActivityAdviceCheck(long threadChannelId, long authorId) {
+        logger.debug("Executing no activity advice check for thread {} by author {}.",
+                threadChannelId, authorId);
+
+        ThreadChannel threadChannel = jda.getThreadChannelById(threadChannelId);
+        if (threadChannel == null) {
+            logger.debug(
+                    "Channel for no activity advice check seems to be deleted (thread {} by author {}).",
+                    threadChannelId, authorId);
+            return;
+        }
+
+        if (threadChannel.isArchived()) {
+            logger.debug(
+                    "Channel for no activity advice check is archived already (thread {} by author {}).",
+                    threadChannelId, authorId);
+            return;
+        }
+        if (threadChannel.getHistory()
+            .getRetrievedHistory()
+            .stream()
+            .map(Message::getAuthor)
+            .map(User::getIdLong)
+            .anyMatch(id -> id == authorId)) {
+            MessageEmbed embed = HelpSystemHelper.embedWith(
+                    """
+                            Hey there {}👋 It has been five minutes and you have not shown any activity in this thread.
```

Comment:
hard coded the time. either use a variable or just avoid mentioning it explicity. maybe

> Hey there John 👋 It has been a bit after you created this thread and you still did not share any details of your question.
> Helpers have seen your question already and are just waiting for you to elaborate on your problem and provide detailed information on it 👌

Reply:
Done